### ISSUE TITLE
[Question] strange loss and repeat valid image

### ISSUE THREAD

**Issue by Pevernow** [frutiemax92/YAT#7]
![image](https://github.com/user-attachments/assets/6dd28d59-a100-4e61-a14d-b57059b5f4dc)
I am training the 1024x version of the multi-language version, with the default learning rate.

The training set has 25,000 images, so I don't think it's easy to overfit.

But as you can see, the loss value is as confusing as a roller coaster.
The five validation images generated are exactly the same, from 0 to 4000 steps

**Comment by Pevernow** [comment]
config:
```
# cloudflare settings
#r2_endpoint: oknjaoinoufew
#r2_access_key: oijoifejnowfeifew
#r2_secret_key: ojwefoifjoifewjefw
#r2_bucket_name: ofiwejfweoijfweqoinfew
#r2_tar_files: 
#  - mwefokfweokfawjeoiwfean
#  - rgejiweorgjgroirewjg
#  - jiofwejfeoifewq
#  - jewfoikjoieqjwhefwq
urls: ["/home/linjl/zzc/dataset/output.tar"]

# training parameters
#n_gpu: 1
#num_processes: 1
batch_size: 1
learning_rate: 1.0e-5
steps: 50000
num_steps_per_validation: 1000
bfloat16: true
gradient_accumulation_steps: 1
validation_prompts:
  - "draw a girl"

# model parameters
pretrained_pipe_path: Efficient-Large-Model/Sana_1600M_1024px_MultiLing_diffusers
#pretrained_pipe_path: Efficient-Large-Model/Sana_600M_512px_diffusers
low_ram: true
```

**Comment by frutiemax92** [comment]
Try with a bigger batch size, you won't get good results with a batch size of 1 and it will stabilize the loss. It is normal at some degree that the loss is not that stable.

**Comment by Pevernow** [comment]
@frutiemax92 
What size of batch size do you think I should choose to handle training with around 2v5 images?
